client/view/Base/colors.css

```diff
@@ -10,6 +10,7 @@
   --text-secondary: oklch(23% 0 0 / 65%);
   --text-underline: oklch(23% 0 0 / 40%);
   --text-underline-hover: oklch(23% 0 0);
+  --text-dark-orange: oklch(72.5% 0.16 53);
```

Comment:
Still explain how it looks like, not the meaning. For instance, you can use `--text-warning`